src/navigators/createBottomTabNavigator.js

```diff
@@ -81,6 +83,7 @@ class TabNavigationView extends React.PureComponent<Props, State> {
     return (
       <TabBarComponent
         {...tabBarOptions}
+        onResize={tabBarHeight => this.setState({ tabBarHeight })}
```

Comment:
We should probably check if the height actually changed before updating the state. I have seen `onLayout` fire unnecessarily before when nothing changed. Also in our case, width might change which will fire `onLayout`, but we only deal with height.